### PR TITLE
Switch to new clickhouse adapter (ch/chto)

### PR DESCRIPTION
**Pull request by aerosol**
### Changes

This PR makes a switch from [clickhousex](https://github.com/plausible/clickhousex) + [ecto_clickhouse](https://github.com/plausible/clickhouse_ecto) to [ch](https://github.com/ruslandoga/ch) + [chto.](https://github.com/ruslandoga/chto)
Additionally it adapts the current codebase to work well with the new adapter.

### Tests
- [x] Automated tests have been added

### Changelog
- [x] This PR does not make a user-facing change

### Documentation
- [x] This change does not need a documentation update

### Dark mode
- [x] This PR does not change the UI

### CODE REVIEW

File: lib/plausible/site/admin.ex [hunk 1]
```diff
@@ -87,11 +87,13 @@ defmodule Plausible.SiteAdmin do
     to_site = Repo.get_by(Plausible.Site, domain: params["domain"])
 
     if to_site do
+      opts = [timeout: 30_000, command: :insert_select]
```
Comment (ruslandoga) [inline lib/plausible/site/admin.ex:90]: I'll need to remove `command: :insert_select` here, it's no longer necessary.

**Review by ukutaht — APPROVED** [review]

File: lib/plausible/site/admin.ex [hunk 2]
```diff
@@ -87,11 +87,13 @@ defmodule Plausible.SiteAdmin do
...
+
       event_q = event_transfer_query(from_site.domain, to_site.domain)
-      {:ok, _} = Ecto.Adapters.SQL.query(Plausible.ClickhouseRepo, event_q, [], timeout: 30_000)
+      {:ok, _} = Ecto.Adapters.SQL.query(Plausible.ClickhouseRepo, event_q, [], opts)
 
       session_q = session_transfer_query(from_site.domain, to_site.domain)
-      {:ok, _} = Ecto.Adapters.SQL.query(Plausible.ClickhouseRepo, session_q, [], timeout: 30_000)
+      {:ok, _} = Ecto.Adapters.SQL.query(Plausible.ClickhouseRepo, session_q, [], opts)
```
Comment (ruslandoga) [inline lib/plausible/site/admin.ex:96]: I've opened https://github.com/plausible/analytics/pull/2776 to revert these changes.

**Review by ruslandoga — APPROVED** [review]
